issue found on: analyser/aggregator.go:186:1: paramTypeCombine

an issue has been detected in source code: ./analyser/aggregator.go:186:1: paramTypeCombine: func(colorizer aurora.Aurora, entries []AggregatorLogEntry, notRead []AggregatorLogEntry) could be replaced with func(colorizer aurora.Aurora, entries, notRead []AggregatorLogEntry)

--- a/analyser/aggregator.go
+++ b/analyser/aggregator.go
@@ -189,7 +189,7 @@ func printInfoForMessageWithOffset(colorizer aurora.Aurora, entries []Aggregator
 	}
 }
 
-func printConsumedEntries(colorizer aurora.Aurora, entries []AggregatorLogEntry, notRead []AggregatorLogEntry) {
+func printConsumedEntries(colorizer aurora.Aurora, entries, notRead []AggregatorLogEntry) {
 	for i, entry := range notRead {
 		printConsumedEntry(colorizer, i+1, entry)
 		printErrorsForMessageWithOffset(colorizer, entries, entry.Offset)

